src/electionguardtest/key_ceremony_helper.py

```diff
@@ -0,0 +1,106 @@
+from typing import List
+
+from electionguard.guardian import Guardian
+from electionguard.key_ceremony import CeremonyDetails, ElectionPartialKeyVerification
+from electionguard.key_ceremony_mediator import GuardianPair, KeyCeremonyMediator
+
+from .identity_encrypt import identity_auxiliary_decrypt, identity_auxiliary_encrypt
+
+
+class KeyCeremonyHelper:
```

Comment:
i think this is considerably more readable than what we had before

Reply:
im also of two minds on files like this, which could clearly be useful in an optional package/module for library consumers and then the actual "test" generators pieces, which are only useful for the tests.  In c++ i create a separate library called 'mocks' that includes all the test generators, but im not quire sure what to do with files like this long term.  perhaps the answer is that we intend to refactor into component modules at some point and we handle this then.

Reply:
you and me both. I really think this one is slowly building out to a package. 

Purpose 1: Mock the steps in an election for easy 
Purpose 2: Assist in generation and complicated flow for testing library